mlflow/server/js/src/common/utils/FeatureUtils.ts

```diff
@@ -52,24 +58,28 @@ export const shouldEnableUnifiedChartDataTraceHighlight = () => true;
 export const shouldUseRegexpBasedAutoRunsSearchFilter = () => true;
 export const shouldUseRunRowsVisibilityMap = () => true;
 export const isUnstableNestedComponentsMigrated = () => true;
+export const shouldUsePredefinedErrorsInExperimentTracking = () => true;
 
 /**
  * Determines if logged models UI (part of model-centric IA shift) is enabled
  */
 export const isExperimentLoggedModelsUIEnabled = () => true;
+export const isLoggedModelsFilteringAndSortingEnabled = () => true;
 export const isRunPageLoggedModelsTableEnabled = () => isExperimentLoggedModelsUIEnabled();
 
 /**
  * Determines if evaluation results online monitoring UI is enabled
  */
 export const isExperimentEvalResultsMonitoringUIEnabled = () => false;
+export const shouldUseRenamedUnifiedTracesTab = () => true;
 
 /**
  * Flags enabling fetching data via GraphQL for particular views:
  */
-export const shouldEnableGraphQLRunDetailsPage = () => true;
+export const shouldEnableGraphQLRunDetailsPage = () => false;
 export const shouldEnableGraphQLSampledMetrics = () => false;
 export const shouldEnableGraphQLModelVersionsForRunDetails = () => false;
+export const shouldRerunExperimentUISeeding = () => false;
```

Comment:
@hubertzub-db let me know if there are any other flags that should be enabled.

Reply:
I think we should leave `shouldEnableGraphQLRunDetailsPage` on `true`, I believe it should work fine by now and it unblocks few new UI features related to logged models on run details page

Reply:
Got it, I'll set it to True.